lvgl.h

```diff
@@ -16,7 +16,7 @@ extern "C" {
 #define LVGL_VERSION_MAJOR 9
 #define LVGL_VERSION_MINOR 0
 #define LVGL_VERSION_PATCH 0
-#define LVGL_VERSION_INFO "dev"
```

Comment:
In line with [this comment](https://github.com/lvgl/lvgl/issues/4926#issuecomment-1891789674) please use the `"rc"` tag for now.

Reply:
Done